framework/src/modules/nft/cc_commands/cc_transfer.ts

```diff
@@ -67,18 +67,15 @@ export class CrossChainTransferCommand extends BaseCCCommand {
 			throw new Error('NFT is not native to either the sending chain or the receiving chain');
 		}
 
-		let nft;
-		try {
-			nft = await this._method.getNFT(getMethodContext(), nftID);
-		} catch (error) {
-			/* empty */
-		}
+		const nftStore = this.stores.get(NFTStore);
+		const nftExists = await nftStore.has(getMethodContext(), nftID);
```

Comment:
is this equivalent of `getNFT`? because there is 2 path in the getNFT which can be not found

Reply:
It is not equivalent however per the LIP it only needs the 1st path i.e. if entry in NFT store is present, so should be correct now.